frontend-react/src/contexts/SessionStorageContext.tsx

```diff
@@ -0,0 +1,34 @@
+import { createContext, FC, useMemo } from "react";
+
+import useSessionStorage, { SessionStore } from "../hooks/UseSessionStorage";
+
+interface ISessionContext {
+    values: SessionStore;
+    updateSessionStorage: (store: Partial<SessionStore>) => void;
+}
+
+export const SessionStorageContext = createContext<ISessionContext>({
+    values: {},
+    updateSessionStorage: (store: Partial<SessionStore>) => {
+        console.log("to please SonarCloud");
+    },
+});
+
+const SessionProvider: FC = ({ children }) => {
+    const { values, updateSessionStorage } = useSessionStorage();
+
+    const payload = useMemo(() => {
+        return {
+            values: values,
+            updateSessionStorage: updateSessionStorage,
```

Comment:
I _think_ I can just not destructure the `useSessionStorage` call and that can be the provider value?